xerxes.md

```diff
@@ -1,6 +1,6 @@
 <popup :custom-text="`<p><a href='https://mpi-eva-archaeogenetics.github.io/comp_human_adna_book/fstats.html'>Introduction to F3- and F4-Statistics</a> by Stephan Schiffels: An explanation of F-Statistics and how to run them with xerxes</p>`"></popup>
```

Comment:
xerxes.pdf was not affected by these changes?

Reply:
Hmmm, what is this? I will check.

Reply:
I'm actually not following what you're getting at? That line (Nr. 1) did not change, right? This whole part above the first heading is removed by the sed-command for the PDF.

Reply:
Sorry - that was not clear. I saw that xerxes.pdf (and actually also janno_details.pdf) did not change, but trident.pdf did. I was just wondering why some of the changes in the .md files had no consequences in the .pdf files but others.

Reply:
But the [GitHub Action](https://github.com/poseidon-framework/poseidon-framework.github.io/actions/runs/10490127511/job/29056190508) looks fine. So I guess they did get rendered properly and just happend to be identical to the previous version.

Reply:
Right. OK.